### ISSUE TITLE
TestFiles depends on unmarked external server

### ISSUE THREAD

**Issue by dopplershift** [Unidata/thredds#223]
TestFiles relies upon accessing dods.mbari.org; however, this is not protected by an assume, which means it can cause a test failure.

**Comment by DennisHeimbigner** [comment]
I have been rethinking this.

IMO, we should rethink the use of ExternalServer.java.
The reason is that most of the servers we reference
are used only for testing or are not under the control
of Unidata (like mbari.org).

So, the only way we have to detect server failure in a timely
fashion is through the failure of a testcase under Jenkins
or Travis. As it is, using ExternServer in effect hides the
information that a server has failed by silently not running
some tests.

Thredds.ucar.edu is one exception to this for the simple reason
that it is used by enough other users that we will hear about it
if it fails. This is not true for the servers used in testing only.

Bottom line: I am closing this issue without any modifications.

**Comment by cwardgar** [comment]
Resolved by #236.
